src/openrct2/ride/Ride.cpp

```diff
@@ -2899,9 +2899,8 @@ static void ride_music_update(Ride* ride)
         return;
     }
 
-    int32_t x = ride->stations[0].Start.x * 32 + 16;
-    int32_t y = ride->stations[0].Start.y * 32 + 16;
-    int32_t z = ride->stations[0].Height * 8;
+    CoordsXYZ rideCoords{ ride->stations[0].Start.x * 32 + 16, ride->stations[0].Start.y * 32 + 16,
```

Comment:
This can be achieved much more cleanly by converting `ride->stations[0].Start` to `CoordsXYZ` first, then calling `ToTileCentre()`.

Reply:
I was unsure if I should have omitted both the multiplication and sum or just the sum, please double check on my latest commit.

Reply:
I meant both. After all, it is a bog standard conversion from TileCoords to Coords.

There should be a method to create `Coords` from `TileCoords` (and if there isn't, there should be one).

Reply:
Done. These weren't really `TileCoords` and I didn't want to change them here, as they might propagate a lot, but though it doesn't look ideal, I created the method and it's cleaner anyway. Thanks for the suggestion!